src/plugin/sdl/sdl.c

```diff
@@ -786,7 +845,15 @@ static void SDL_handle_events(void)
 	if (config.X_background_pause && !dosemu_user_froze)
 	  freeze_dosemu();
 	break;
+
       case SDL_WINDOWEVENT_RESIZED:
+#if defined(HAVE_SDL2_TTF) && defined(HAVE_FONTCONFIG)
+        if (vga.mode_class == TEXT && !use_bitmap_font) {
+          SDL_resize_to_nearest_fontsize(event.window.data1, event.window.data2);
```

Comment:
I still dont understand this part...
Isn't it a winsize, not fontsize?

Reply:
Takes the resized window sizes and adjusts them to be a multiple of character cell dimensions and then resizes window again to the result.

Reply:
Yes, but the name is confusing.
Something like `SDL_resize_to_nearest_winsize` should
be better. But I don't think second-guess the resize event
is a good idea, so how about resizing precisely to the needed
size, and left some border around as adjustment?

Reply:
latest commit removes this name and the window resize.